CMakeLists.txt

```diff
@@ -15,6 +15,35 @@ message(STATUS "Building the MPACT compiler at ${MPACT_SOURCE_DIR} (into ${MPACT
 
 set(MPACT_PYTHON_PACKAGES_DIR "${MPACT_BINARY_DIR}/python_packages")
 
+#-------------------------------------------------------------------------------
+# Configure out-of-tree vs in-tree build
+#-------------------------------------------------------------------------------
+
+if(CMAKE_SOURCE_DIR STREQUAL CMAKE_CURRENT_SOURCE_DIR)
+  message(STATUS "Torch-MLIR out-of-tree build.")
+  message(FATAL_ERROR "TODO")
+else()
+  message(STATUS "Torch-MLIR in-tree build.")
+  # In-tree build with LLVM_EXTERNAL_PROJECTS=torch-mlir
+
+  option(MLIR_ENABLE_BINDINGS_PYTHON "Enables MLIR Python Bindings" OFF)
```

Comment:
ON?

Reply:
I think it is off by default for LLVM, and we used `MLIR_ENABLE_BINDINGS_PYTHON=On` in our readme too.

Reply:
Oh, it is the default only. The reason I asked is because in my own setup I made a few assumptions on having bindings on. But I don't understand why we need the define option here again, since it is also a MLIR option.

But perhaps using MPACT_ ... helps?

Reply:
TBH, I copied this from torch-mlir, maybe they simply tried to import the option. ;-)